disnake/channel.py

```diff
@@ -448,13 +448,80 @@ async def edit(
             # the payload will always be the proper channel payload
             return self.__class__(state=self._state, guild=self.guild, data=payload)  # type: ignore
 
-    @utils.copy_doc(disnake.abc.GuildChannel.clone)
     async def clone(
-        self, *, name: Optional[str] = None, reason: Optional[str] = None
+        self,
+        *,
+        name: Optional[str] = None,
+        topic: Optional[str] = None,
+        position: Optional[int] = None,
+        nsfw: Optional[bool] = None,
+        category: Optional[Snowflake] = None,
+        slowmode_delay: Optional[int] = None,
+        type: Optional[ChannelType] = None,
+        default_auto_archive_duration: Optional[AnyThreadArchiveDuration] = None,
+        flags: Optional[ChannelFlags] = None,
+        reason: Optional[str] = None,
     ) -> TextChannel:
+        """|coro|
+
+        Clones this channel. This creates a channel with the same properties
+        as this channel.
+
+        You must have :attr:`.Permissions.manage_channels` permission to
+        do this.
+
+        .. versionadded:: 1.1
+
+        .. versionchanged:: 2.7
```

Comment:
This should mention what changed  - a short summary is usually enough, something along the lines of ``... added new parameters in addition to name and reason``